### PR TITLE
A couple process things

### PR DESCRIPTION
**Pull request by jlevy**
More on reviewing a PR

Drop the Airtable link since it's better just to use Slack

Also trimmed some trailing whitespace.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -28,7 +33,31 @@ If you’ve found this guide useful, you have many ways to help:
...
+3.	Can the same information be conveyed with more succinctly or more clearly?
+4.	Copy editing: Does it match the [writing style](#writing-style) and [conventions](#writing-conventions) (including emoji symbols, punctuation, and grammar)?
+5.	Are there any other parts of the guide that should be updated at the same time? (Examples are adding new sections to the table of contents, adding new internal links, rows to tables, etc.)
+6.	Are there additional improvements that should be filed as issues?
+
+### Editorial Process
+
+-	Roles:
+	-	**Project leads:** Own overall quality of the Guide, direction, and process.
+	-	**Editors:** Contributors own specific sections or aspects of the Guide, reviewing PRs and/or writing. requires expert knowledge.
```
Comment (ThanosBaskous) [inline CONTRIBUTING.md]: Capitalization: "Requires"

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -28,7 +33,31 @@ If you’ve found this guide useful, you have many ways to help:
...
+	-	**Experts:** People with expert knowledge in various areas, who have agreed to review or help on demand with tougher questions or PRs.
```
Comment (ThanosBaskous) [inline CONTRIBUTING.md]: Punctuation: I don't think the comma is necessary.

File: README.md [hunk 3]
```diff
@@ -93,9 +93,9 @@ Before using the guide, please read the [**license**](#license) and [**disclaime
...
+[![Slack Chat](https://img.shields.io/badge/Chat-Slack-ff69b4.svg "Join us. Anyone is welcome!")](https://og-aws.slack.lexikon.io/)
 
-Please help by [**joining the Slack channel**](https://og-aws.slack.lexikon.io/) to talk about AWS (anyone is welcome, even if you only have questions!), [**submitting a question**](https://airtable.com/shrXZ61VrovWfXYBg), or [**contributing to the guide**](CONTRIBUTING.md). This guide is *open to contributions*, so unlike a blog, it can keep improving. Like any open source effort, we combine efforts but also review to ensure high quality.
+Please help by [**joining the Slack channel**](https://og-aws.slack.lexikon.io/) (we like to talk about AWS in general, even if you only have questions — discussion helps the community and guides what contributors) and [**contributing to the guide**](CONTRIBUTING.md). This guide is *open to contributions*, so unlike a blog, it can keep improving. Like any open source effort, we combine efforts but also review to ensure high quality.
```
Comment (ThanosBaskous) [inline README.md]: Missing word or extra word? "discussion helps the community and guides what contributors"
"discussion helps the community and guides what contributors cover"
"discussion helps the community and guides contributors"

**Comment by ThanosBaskous** [comment]
👍 Thanks @jlevy

**Comment by jlevy** [comment]
Thanks for the copy editing. Guess I'm tired. :P Slightly tweaked one more thing.

**Pull request merged**